src/accessibility.cs

```diff
@@ -1,9 +1,96 @@
 using System;
+using CoreGraphics;
 using Foundation;
 using ObjCRuntime;
 
 namespace Accessibility {
 
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AXCategoricalDataAxisDescriptor : AXDataAxisDescriptor
+	{
+		[Export ("categoryOrder", ArgumentSemantic.Copy)]
+		string[] CategoryOrder { get; set; }
+
+		[Export ("initWithTitle:categoryOrder:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string title, string[] categoryOrder);
+
+		[Export ("initWithAttributedTitle:categoryOrder:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (NSAttributedString attributedTitle, string[] categoryOrder);
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	interface AXChart
+	{
+		[Abstract]
+		[NullAllowed, Export ("accessibilityChartDescriptor", ArgumentSemantic.Strong)]
+		AXChartDescriptor AccessibilityChartDescriptor { get; set; }
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[Native]
+	public enum AXChartDescriptorContentDirection : long
+	{
+		LeftToRight = 0,
+		RightToLeft,
+		TopToBottom,
+		BottomToTop,
+		RadialClockwise,
+		RadialCounterClockwise,
+	}
+
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0)]
+	[BaseType (typeof(NSObject))]
```

Comment:
Same as what @spouliot mentioned.

Reply:
Nope, this is an actual object `@interface AXChartDescriptor : NSObject` not a delegate so BaseType is fine here